packages/commons/src/conditionals/conditionals.ts

```diff
@@ -9,19 +9,57 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 
-import { JSONSchemaType } from 'ajv'
 import { z } from 'zod'
 import { ActionFormData, EventDocument } from '../events'
 
+/** @knipignore */
+export type JSONSchema = {
+  readonly __nominal__type: 'JSONSchema'
+}
+
+export function defineConditional(schema: any) {
+  return schema as JSONSchema
+}
```

Comment:
I made this a "nominal" type so that we wouldn't have to use any. Effectively it's a named "unknown" type you cannot look inside of but it still verifies right parameter is being sent to a right function